src/applications/disability-benefits/view-payments/sass/responsive-table.scss

```diff
@@ -0,0 +1,60 @@
+table.responsive {
+  font-family: "Open Sans", sans-serif;
+  border-collapse: collapse;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+
+  th {
+    vertical-align: bottom;
+  }
+
+  td {
+    word-wrap: break-word;
+    vertical-align: top;
+  }
+}
+
+@media screen and (max-width: $medium-screen) {
+
+  table.responsive {
+    border: 0;
+    thead {
+      border: none;
+      clip: rect(0, 0, 0, 0);
+      height: 1px;
+      margin: -1px;
+      overflow: hidden;
+      padding: 0;
+      width: 1px;
+    }
+
+    tr {
+      border-bottom: 1px solid #757575;
+
+      &:first-child {
+        border-top: 1px solid #757575 !important;
```

Comment:
I think we'll see warnings about this unless a predefined CSS variable is used here.

Reply:
Changed to variables